src/main/java/com/matthewglover/tictactoe/core/MiniMax.java

```diff
@@ -1,26 +1,37 @@
 package com.matthewglover.tictactoe.core;
 
 public abstract class MiniMax {
-    private static final int DRAW_SCORE = 0;
-    private static final int WINNING_SCORE = 10;
+    protected final int DRAW_SCORE = 0;
+    private final int winningScore;
     protected final Game game;
```

Comment:
protected for instance variables should probably be avoided. Figuring out where the value is mutated can be difficult, especially as your inheritance tree expands.